modules/virt-configuring-storage-class-bootsource-update.adoc

```diff
@@ -76,4 +79,4 @@ $ oc patch storageclass <current_default_storage_class> -p '{"metadata": {"annot
 ----
 $ oc patch storageclass <new_storage_class> -p '{"metadata":{"annotations":{"storageclass.kubernetes.io/is-default-class":"true"}}}' <1>
 ----
-<1> Replace `<new_storage_class>` with the `storageClassName` value that you added to the `HyperConverged` CR.
\ No newline at end of file
+<1> Replace `<new_storage_class>` with the `storageClassName` value that you added to the `HyperConverged` CR.
```

Comment:
I think we need to edit some things at the beginning of the doc as well.

`Specify a new default storage class in the `HyperConverged` custom resource (CR).` - does not sound right. We need either a default storage class, or a specified in the HyperConverged CR, which should override the default.

In addition to that, since 4.15 we have a default-virt annotation
`storageclass.kubevirt.io/is-default-virt-class: "true"`
that will be preferred over the kubernetes default annotation
`storageclass.kubernetes.io/is-default-class":"true"`

This is a bit information about this annotation: https://github.com/kubevirt/community/blob/main/design-proposals/default-virt-storage-class.md
(please pay attention, in the design proposal there's an outdated annotation (different wording), but here I pasted the correct one)

Reply:
@ousleyp Are you awar of this? It seem like a matter that should be addressed on its own. 

> In addition to that, since 4.15 we have a default-virt annotation
> storageclass.kubevirt.io/is-default-virt-class: "true"
> that will be preferred over the kubernetes default annotation
> storageclass.kubernetes.io/is-default-class":"true"

Reply:
virt sc was handled in https://github.com/openshift/openshift-docs/pull/70639

Reply:
Right, it was addressed separately, but if we have a section about the default storage class for the boot source images - we should mention here both virt default and kubernetes default.

virt default will be preferred, but it will not show the **(default)** part, when we do `oc get storageclass`

Reply:
@mburke5678 so will we address this as a separate issue?